BrainStormApp/Helpers/UIHelper.swift

```diff
@@ -37,7 +37,7 @@ class UIHelper {
     }
     
     class func showCategoryAction(from viewController: UIViewController,
-                                  completionHandler: @escaping ((Category?) -> Void)) {
+                                  completionHandler: @escaping ((Category?,Bool) -> Void)) {
```

Comment:
We need to tell what cause of nil category - cancel button or empty category, if we click cancel then category shouldn't change to empty. Perhaps not the best way but it works.